electroncash/wallet.py

```diff
@@ -1874,7 +1874,7 @@ def get_default_change_addresses(self, count):
                 result.extend(new_addrs)
             return result
 
-    def make_unsigned_transaction(self, inputs, outputs, config, fixed_fee=None, change_addr=None, sign_schnorr=None):
+    def make_unsigned_transaction(self, inputs, outputs, config, fee=None, change_addr=None, sign_schnorr=None):
```

Comment:
Please do not rename this 4th arg. Sadly, plugins may rely on this arg's name here.  Due to the plugin system we are doomed to never really be able to randomly change things unless we can be sure none of the popular external plugins would break.